client/app/host/TeamInspector.tsx

```diff
@@ -0,0 +1,116 @@
+'use client';
+
+import {
+    Select,
+    SelectContent,
+    SelectItem,
+    SelectTrigger,
+    SelectValue,
+} from '@/components/ui/select';
+import { Send, Wifi, WifiOff } from 'lucide-react';
+import { useTeamsAtom, useCurrentTeam } from '@/lib/host-state';
+import { Separator } from '@/components/ui/separator';
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+import { Button } from '@/components/ui/button';
+import { Input } from '@/components/ui/input';
+import { toast } from '@/hooks/use-toast';
+
+export default function TeamInspector() {
+    const { teamList } = useTeamsAtom();
+    const { selectedTeam, setSelectedTeam } = useCurrentTeam();
+
+    return (
+        <div className="flex h-full w-full flex-col items-center">
+            <Select
+                value={selectedTeam?.name || ''}
+                onValueChange={(value) => {
+                    const team = teamList.find((t) => t.name === value);
+                    if (team) setSelectedTeam(team);
+                }}
+            >
+                <SelectTrigger className="flex w-fit">
+                    <SelectValue placeholder="Select A Team">
+                        {selectedTeam ? (
+                            <span className="flex gap-1">
+                                {selectedTeam.status ? (
+                                    <Wifi className="text-green-500" />
+                                ) : (
+                                    <WifiOff className="text-gray-300 dark:text-gray-500" />
+                                )}
+                                {selectedTeam.name}
+                            </span>
+                        ) : (
+                            'Select A Team'
+                        )}
```

Comment:
This is not necessary.  The Select will automatically use the value in the select item.